scheduler/test/cook/test/kubernetes/api.clj

```diff
@@ -12,48 +12,82 @@
                                                 V1VolumeMount)))
 
 (deftest test-get-consumption
-  (testing "correctly computes consumption for a single pod"
+  (testing "correctly computes consumption for a single pod without gpus"
+
     (let [pods [(tu/pod-helper "podA" "hostA" {:cpus 1.0 :mem 100.0})]
           node-name->pods (api/pods->node-name->pods pods)]
       (is (= {"hostA" {:cpus 1.0
-                       :mem 100.0}}
+                       :mem 100.0
+                       :gpus {}}}
              (api/get-consumption node-name->pods)))))
 
-  (testing "correctly computes consumption for a pod with multiple containers"
+  (testing "correctly computes consumption for a single pod with gpus"
+
+    (let [pods [(tu/pod-helper "podA" "hostA" {:cpus 1.0 :mem 100.0 :gpus "2" :gpu-model "nvidia-tesla-p100"})]
+          node-name->pods (api/pods->node-name->pods pods)]
+      (is (= {"hostA" {:cpus 1.0
+                       :mem 100.0
+                       :gpus {"nvidia-tesla-p100" 2}}}
+             (api/get-consumption node-name->pods)))))
+
+  (testing "correctly computes consumption for a pod with multiple containers without gpus"
     (let [pods [(tu/pod-helper "podA" "hostA"
-                               {:cpus 1.0 :mem 100.0}
+                               {:cpus 1.0 :mem 100.0 :gpus "0"}
```

Comment:
is :gpu's supposed to be a string, here and other tests. That seems very surprising!

Reply:
Yeah it is surprising but that is how gpus were represented in the pod yaml, which is why we are doing the same thing in the unit tests.